lib/Server.js

```diff
@@ -1,35 +1,78 @@
 'use strict';
 
+const eventbus = require('probo-eventbus');
 const through2 = require('through2');
 
 class Server {
   constructor(options) {
-    this.consumer = options.consumer;
-  }
-  storeBuildInDatabase(build, done) {
-    // Do stuff
-    let error = null;
-    done(null);
-  }
+    this.db = options.db;
+    this.config = options.config;
+
+    this.consumer = new eventbus.plugins['Kafka'].Consumer({
```

Comment:
The other open source projects make the eventbus plugin configurable. We should do that here just for consistency.

See here:
https://github.com/ProboCI/probo/blob/master/defaults.yaml#L186
https://github.com/ProboCI/probo/blob/ef8148938e8c9ea37be12d1a1e8b4218a9f42cb6/lib/ContainerManager.js#L169